build.assets/tooling/cmd/build-apt-repos/s3logger.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type s3logger struct{}
```

Comment:
Why is this needed?

Reply:
This maps the s3sync logging functions to logrus so that they match the rest of the logging framework's settings. Their docs outline it here: https://github.com/seqsense/s3sync#sets-the-custom-logger

Reply:
Let's add this explanation here as a godoc comment.

Reply:
Added pending push